OPLS.py

```diff
@@ -77,7 +78,12 @@ def OPLSMin(ifile,opttype):
             help = "Name of the force field type to be used for minimization",
             type = "string",
             dest = 'opttype')
+    
+    parser.add_option('-d','--outdir',
+            help = "Directory of the output location",
+            type = "string",
+            dest = 'outdir')
     (opt, args) = parser.parse_args()
-    OPLSMin(opt.ifile,opt.opttype)
+    OPLSMin(opt.ifile,opt.opttype,opt.outdir)
```

Comment:
ifile is supposed to be a file or a directory? Your method OPLSMin treats it like a file, but the input sounds like it is a directory.